src/main/java/org/fundacionjala/coding/abel/BankOCR.java

```diff
@@ -0,0 +1,153 @@
+
+    /**
+     *
+     */
+    private BankOCR() {
+
+    }
+
+    /**
+     * Read the entry.
+     * @param entry is the parameter
+     * @return String
+     */
+    public static String convertEntryToNumber(String entry) {
+        stringNumbers = new ArrayList<>();
+        for (int i = 0; i < TOTAL_CHARS; i++) {
+            StringBuilder stringBuilder = new StringBuilder();
+            stringBuilder.append(entry.substring(FIRST_LINE_AT_ZERO + i * CHAR_LENGTH,
+                    FIRST_LINE_AT_THREE + i * CHAR_LENGTH));
+            stringBuilder.append(entry.substring(SECOND_LINE_AT_ZERO + i * CHAR_LENGTH,
+                    SECOND_LINE_AT_THREE + i * CHAR_LENGTH));
+            stringBuilder.append(entry.substring(THIRD_LINE_AT_ZERO + i * CHAR_LENGTH,
+                    THIRD_LINE_AT_THREE + i * CHAR_LENGTH));
+            stringNumbers.add(stringBuilder.toString());
+        }
+        return stringNumbers.stream()
+                .map(number -> getNumberValue(number))
+                .collect(Collectors.joining());
+    }
+
+    /**
+     *
+     * @param string parameter
+     * @return String
+     */
+    private static String getNumberValue(String string) {
+        String result = ALL_NUMBERS.entrySet().stream()
+                .filter(map -> string.equals(map.getKey()))
+                .map(map -> String.valueOf(map.getValue()))
+                .collect(Collectors.joining());
+
+        return (result.equals("")) ? "?" : String.valueOf(result);
+    }
+
+    /**
+     *
+     * @param fileName param
+     * @return String
+     */
+    public static String readFile(String fileName) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        try {
+            Stream<String> stream = Files.lines(Paths.get(fileName));
+            stream.forEach(st -> stringBuilder.append(st));
+
+            return stringBuilder.toString();
+        } catch (IOException e) {
+            System.out.println(e);
+        }
+        return null;
+    }
+
+    /**
+     *
+     * @param accountNumber param
+     * @return Boolean
+     */
+    public static boolean validateAccountNumber(String accountNumber) {
+        if (accountNumber.contains("?")) {
+            return false;
+        }
+        char[] charArray = new StringBuilder(accountNumber).reverse().toString().toCharArray();
+        int sum = 0;
+        for (int i = 0; i <= charArray.length - 1; i++) {
+            int intValue = Integer.parseInt(String.valueOf(charArray[i]));
+            sum += intValue * (i + 1);
+        }
+        return sum % ELEVEN == 0;
+    }
+
+    /**
+     *
+     * @param entryToNumber param
+     * @return String
+     */
+    public static String lineOutPut(String entryToNumber) {
+        return (validateAccountNumber(entryToNumber)) ? entryToNumber : ((entryToNumber.contains("?"))
+                ? String.join("", entryToNumber, " ILL")
+                : String.join("", entryToNumber, " ERR"));
```

Comment:
I this case the ternary operator is to complex.
Please try to change it in a simple way.